### PR TITLE
Remove barrier specific tracking

### PR DESCRIPTION
**Pull request by remybach**
https://trello.com/c/Hme2a2xv/1286-5-move-barrier-tracking-from-n-ui-to-next-product

🐿 v2.12.3

### CODE REVIEW

File: components/n-ui/tracking/ft/index.js [hunk 1]
```diff
@@ -126,27 +126,14 @@ const oTrackingWrapper = {
...
 
-			if (barrierType) {
-				const customTracking = JSON.parse(barrierType.getAttribute('data-tracking-object'));
-
-				broadcast('oTracking.event', Object.assign({
-					category: 'barrier',
-					action: 'view'
-				}, context, customTracking));
+			if (window.FT && window.FT.nUi) {
+				window.FT.nUi.trackingContext = context;
```
Comment (i-like-robots) [inline components/n-ui/tracking/ft/index.js]: I'm guessing this is being set so this data can be picked up from the outside... what will pick it up?

Comment (remybach) [reply components/n-ui/tracking/ft/index.js]: https://github.com/Financial-Times/next-product/pull/752/files#diff-096103d8490b885f9d5037baea6aea80R82 and https://github.com/Financial-Times/next-epaper/pull/35/files#diff-a93ac81418745fbaa6d1e0e4a4171132R20

Comment (i-like-robots) [reply components/n-ui/tracking/ft/index.js]: Could you add a comment to the code with these references? It's scary setting a global and not knowing who or what consumes it!

Comment (remybach) [reply components/n-ui/tracking/ft/index.js]: I've added a more generic comment... direct references tend to get outdated and are unlikely to get updated when this gets used elsewhere. At least there's some indication and a "why" now... hopefully that will suffice?

Comment (i-like-robots) [reply components/n-ui/tracking/ft/index.js]: Thanks!

**Pull request merged**